src/containers/quiz/question-answer/Answer.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  FormControlLabel,
+  FormControlLabelProps,
+  Box,
+  Checkbox,
+  Radio,
+  InputAdornment,
+  SxProps
+} from '@mui/material'
```

Comment:
Use default imports instead because they are better from performance perspective

Reply:
Thanks, used default imports here and in other places